front/src/app/pages/agency-dashboard/MarkPartnersErroredConventionAsHandledFormSection.tsx

```diff
@@ -58,7 +61,21 @@ export const MarkPartnersErroredConventionAsHandledFormSection = ({
             <Input
               label="Id de la convention *"
               nativeInputProps={{
-                ...methods.register("conventionId"),
+                ...conventionIdRegistration,
+                onChange: (event) => {
+                  if (event.currentTarget.value.length >= 32) {
+                    dispatch(
+                      conventionSlice.actions.fetchConventionRequested({
+                        conventionId: event.currentTarget.value,
+                        jwt,
+                      }),
+                    );
+                  }
+                  if (event.currentTarget.value.length === 0) {
+                    dispatch(conventionSlice.actions.clearFetchedConvention());
+                  }
+                  conventionIdRegistration.onChange(event);
+                },
```

Comment:
tu penses quoi de mettre ce target dans une variable ?

```suggestion
                onChange: (event) => {
                  const conventionId = event.currentTarget.value;
                  if (conventionId.length >= 32) {
                    dispatch(
                      conventionSlice.actions.fetchConventionRequested({
                        conventionId,
                        jwt,
                      }),
                    );
                  }
                  if (conventionId.length === 0) {
                    dispatch(conventionSlice.actions.clearFetchedConvention());
                  }
                  conventionIdRegistration.onChange(event);
                },
```